lisa/sut_orchestrator/azure/common.py

```diff
@@ -1904,3 +1916,284 @@ def get_size(disk_type: schema.DiskType, data_disk_iops: int = 1) -> int:
             return iops_dict[min_iops]
         else:
             raise LisaException(f"Data disk type {disk_type} is unsupported.")
+
+
+def get_tenant_id(credential: Any) -> Any:
+    # Initialize the Subscription client
+    subscription_client = SubscriptionClient(credential)
+    # Get the subscription
+    subscription = next(subscription_client.subscriptions.list())
+    return subscription.tenant_id
+
+
+def get_object_id() -> Any:
+    # Define constants
+    graph_api_url = "https://graph.microsoft.com/.default"
+    request_url = "https://graph.microsoft.com/v1.0/me"
+
+    # Get a token for the Microsoft Graph API
+    token_credential = DefaultAzureCredential()
+    token = token_credential.get_token(graph_api_url)
+
+    # Set up the API call headers
+    headers = {
+        "Authorization": f"Bearer {token.token}",
+        "Content-Type": "application/json",
+    }
+
+    # Set a timeout of 10 seconds for the request
+    response = requests.get(request_url, headers=headers, timeout=10)
+
+    if response.status_code == 200:
```

Comment:
The logic can be simpler like

```suggestion
    if response.status_code != 200:
      raise LisaException(
            f"Failed to retrieve user object ID. "
            f"Status code: {response.status_code}. "
            f"Response: {response.text}"
      )

  return response.json().get("id")
```